std/portable/index.js

```diff
@@ -12,6 +12,7 @@ globalScope.ASC_FEATURE_MUTABLE_GLOBAL = false;
 globalScope.ASC_FEATURE_SIGN_EXTENSION = false;
 globalScope.ASC_FEATURE_BULK_MEMORY = false;
 globalScope.ASC_FEATURE_SIMD = false;
+globalScope.ASC_FEATURE_THREADS = false;
```

Comment:
May be better `globalScope.ASC_FEATURE_THREADS = typeof Atomics == "object"`?

Reply:
This will give different output on different environments. That’s bad. We should stick to static flags instead of dynamic. We can enable it by default once the feature is available on every platforms.

Reply:
Yeah, you are right. May be better do this later when all functionality stabilized.